account_invoice_import/views/res_config_settings.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2015-2017 Akretion (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+  <record id="res_config_settings_view_form" model="ir.ui.view">
+    <field name="name">invoice_import.res.config.settings.form</field>
+    <field name="model">res.config.settings</field>
+    <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+    <field name="arch" type="xml">
+      <xpath expr="//h2[contains(text(),'Taxes')]" position="before">
```

Comment:
This expression `<xpath expr="//h2[contains(text(),'Taxes')]" ...` does not behave well with languages other than English. Let me explain better...

Scenario:

- module _account_invoice_import_ is installed
- login as admin
- enable debug mode
- go to "Settings > Configuration > Settings"

The settings panel is correctly displayed.

Now, if you change the language for the admin user (we tried with Dutch language) and you repeat the above steps, you will notice that the settings panel if not displayed at all.
The reason is that the expression `expr="//h2[contains(text(),'Taxes')]"` returns false since the string 'Taxes' is not found.

Could you take a look at it?

Reply:
I'm trying by replacing that line with:

```
    <div data-key="account_invoicing" position="inside">
```
and it works fine to me.

Reply:
Ok will try this (or find a solution that works)